### PR TITLE
Introduce teacher documentation for annotations 

### PR DESCRIPTION
**Pull request by jorg-vr**
This adds an faq about questions and comments

This faq was made in the context of the release of saved annotations: https://github.com/dodona-edu/dodona/pull/4992

### CODE REVIEW

**Comment by bmesuere** [comment]
@jorg-vr I propose rewriting it to these questions

# Questions and comments

## How can I ask a question about my code? <Badge type="tip" text="student" />

(also add an info box that questions might be disabled)

## How do I know if my question has been answered? <Badge type="tip" text="student" />

## How can I enable/disable student questions? <Badge type="tip" text="teacher" />

## How do I know if there are any open questions? <Badge type="tip" text="teacher" />

## How can I reply to a question from a student? <Badge type="tip" text="teacher" />

## How can I comment on a students' submission? <Badge type="tip" text="teacher" />

## How can I store and reuse comments? <Badge type="tip" text="teacher" />

**Comment by bmesuere** [comment]
This would also close #169. There is some content in the issue there that could be used here.

File: en/faq/annotations/index.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+If there are open questions within a course, you will see a notification in the course card on your homepage.
+
+![img.png](./course_card_question.png)
+
+You will also see a notification in the top navigation bar of your course. Click on the notification to go to the page where you can see all the open questions.
+
+![img.png](./course_question_list.png)
+
+If you want to see al open question accross all your courses, you can go to the `questions` page in the navigation sidebar of Dodona.
```
Comment (pdawyndt) [inline en/faq/annotations/index.md:60]: across

File: en/faq/annotations/index.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+## How can I reply to a question from a student? <Badge type="tip" text="teacher" />
+
+![Question on code](./question-on-code.png)
+
+You can choose to type a reply to the question by clicking the input field. While typing, the question will be marked as in progress, to notify other teachers someone is already working on it.
+Finishing the reply will mark the question as answered and sends the student a notification.
+
+You can also directly mark a question as answered. This is often useful if you know the student has already solved the question in another way. Eg. the student has submitted an other solution where the problem is fixed, or you have replied to the question outside of the Dodona framework.
+
+> Note: comments support markdown formatting, so you can use bold, italics, lists, ... You can checkout all markdown features in [this reference](/en/references/exercise-description/#markdown).
```
Comment (pdawyndt) [inline en/faq/annotations/index.md:73]: Make capital/lowercase spelling of Markdown consistent throughout the text. I suggest "Markdown"

File: nl/faq/annotations/index.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+## Hoe kan ik opmerkingen geven op een oplossing van een student? <Badge type="tip" text="lesgever" />
+
+Als lesgever kan je ook opmerkingen geven op de code van je studenten. Dit kan handig zijn om feedback te geven op hun oplossing, of om een fout aan te duiden.
+
+> Als je alle oplossingen van je studenten wil overlopen en feedback wil geven, kan het handig zijn om [een evaluatie aan te maken](/nl/guides/teachers/grading).
+
+Je kan een algemene opmerking toevoegen door op de `Globale opmerking toevoegen` knop te klikken. Je kan ook een stuk code selecteren en op de `Voeg opmerking toe` knop klikken om een opmerking toe te voegen over dat specifieke stuk code.
+
+![add comment](./add-comment.png)
+> Ter zijde: opmerkingen ondersteunen markdown, dus je kan gebruik maken van vet, cursief, lijsten, ... Je kan alle markdown features bekijken in [deze referentie](/nl/references/exercise-description/#markdown).
```
Comment (pdawyndt) [inline nl/faq/annotations/index.md:82]: Markdown

File: nl/faq/annotations/index.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+title: Vragen en opmerkingen
+---
+
+# Vragen en opmerkingen
+
+[[toc]]
+
+## Hoe kan ik vragen stellen over mijn code? <Badge type="tip" text="student" />
+
+> Opmerking: deze functionaliteit is enkel beschikbaar als je lesgever dit heeft ingeschakeld.
```
Comment (niknetniko) [inline nl/faq/annotations/index.md:11]:
```suggestion
::: tip Opmerking
Deze functionaliteit is enkel beschikbaar als je lesgever dit heeft ingeschakeld.
:::
```

File: nl/faq/annotations/index.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+![img.png](./questions_index_page.png)
+
+## Hoe kan ik een vraag beantwoorden? <Badge type="tip" text="lesgever" />
+
+![Question on code](./question-on-code.png)
+
+Je kan een antwoord typen op de vraag door op het invoerveld te klikken. Terwijl je typt, zal de vraag gemarkeerd worden als in behandeling, om andere lesgevers te verwittigen dat iemand er al mee bezig is.
+Wanneer je jouw antwoord afwerkt, zal de vraag gemarkeerd worden als beantwoord en krijgt de student een melding.
+
+Je kan ook direct een vraag markeren als beantwoord. Dit is vaak handig als je weet dat de student de vraag al opgelost heeft op een andere manier. Bijvoorbeeld de student heeft een andere oplossing ingediend waar het probleem opgelost is, of je hebt de vraag beantwoord buiten het Dodona platform.
```
Comment (niknetniko) [inline nl/faq/annotations/index.md:71]:
```suggestion
Je kan ook direct een vraag markeren als beantwoord. Dit is vaak handig als je weet dat de student de vraag al opgelost heeft op een andere manier. Bijvoorbeeld als de student een andere oplossing ingediend heeft waar het probleem opgelost is, of als je de vraag beantwoord hebt buiten Dodona.
```

File: nl/faq/annotations/index.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+
+## Hoe kan ik opmerkingen geven op een oplossing van een student? <Badge type="tip" text="lesgever" />
+
+Als lesgever kan je ook opmerkingen geven op de code van je studenten. Dit kan handig zijn om feedback te geven op hun oplossing, of om een fout aan te duiden.
+
+> Als je alle oplossingen van je studenten wil overlopen en feedback wil geven, kan het handig zijn om [een evaluatie aan te maken](/nl/guides/teachers/grading).
+
+Je kan een algemene opmerking toevoegen door op de `Globale opmerking toevoegen` knop te klikken. Je kan ook een stuk code selecteren en op de `Voeg opmerking toe` knop klikken om een opmerking toe te voegen over dat specifieke stuk code.
```
Comment (niknetniko) [inline nl/faq/annotations/index.md:79]:
```suggestion
Je kan een algemene opmerking toevoegen door op de knop `Globale opmerking toevoegen` te klikken. Je kan ook een stuk code selecteren en op de knop `Voeg opmerking toe` klikken om een opmerking toe te voegen over dat specifieke stuk code.
```

File: nl/faq/annotations/index.md (hunk 3 above)
Comment (niknetniko) [inline nl/faq/annotations/index.md:82]:
```suggestion
> Terzijde: opmerkingen ondersteunen Markdown, dus je kan gebruik maken van vet, cursief, lijsten, ... Je kan alle Markdown features bekijken in [deze referentie](/nl/references/exercise-description/#markdown).
```

File: nl/faq/annotations/index.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+
+Eenmaal je een of meerdere opmerkingen hebt opgeslagen, krijg je een zoekveld te zien wanneer je een nieuwe opmerking schrijft. Selecteer de gewenste opmerking uit de lijst om deze te hergebruiken. Je kan de opmerking nog steeds bewerken voor je deze post.
+
+![Reuse comment](./reuse-annotations.png)
+
+## Waarom vind ik mijn opgeslagen opmerkingen niet terug? <Badge type="tip" text="lesgever" />
+
+Opgeslagen opmerkingen worden gegroepeerd per oefening, cursus en gebruiker. Dit betekent dat je een opmerking kan opslaan voor een oefening, en deze opmerking kan hergebruiken in alle oplossingen van die oefening. 
+
+Als je een opmerking wil hergebruiken in een andere oefening, zal je deze opnieuw moeten opslaan. We onderzoeken momenteel of hoe we dit kunnen verbeteren.
```
Comment (niknetniko) [inline nl/faq/annotations/index.md:103]:
```suggestion
Als je een opmerking wil hergebruiken in een andere oefening, zal je deze opnieuw moeten opslaan. We onderzoeken momenteel hoe we dit kunnen verbeteren.
```

**Review by niknetniko — CHANGES_REQUESTED** [review]

File: en/faq/annotations/index.md [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+
+# Questions and comments
+
+[[toc]]
+
+## How can I ask a question about my code? <Badge type="tip" text="student" />
+
+> Note: this feature is only available if your teacher has enabled it.
+
+![image](./student.ask_questions.png)
```
[the remark this answers is not in the document]
Comment (jorg-vr) [reply en/faq/annotations/index.md:15]: I have made different images for each language now

**Review by niknetniko — APPROVED** [review]

**Review by bmesuere — APPROVED** [review]

**Pull request merged**